### PR TITLE
Update search-the-audit-log-in-security-and-compliance.md

### PR DESCRIPTION
**Pull request by jetkjh**
Please update the documentation as this is confusing for an admin, the options are available in the UI but not functional. 

Please ref the MS BUG:
VSO bug number -1295817

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/search-the-audit-log-in-security-and-compliance.md [hunk 1]
```diff
@@ -54,6 +54,9 @@ Need to find if a user viewed a specific document or purged an item from their m
 
 - User and admin activity in Microsoft PowerApps
 
+  > [!NOTE]
+  >You may see some audited activities available in the Security and Compliance Center Audit log search UI that are not yet functional (such as Forms events). Until the these classifications of events are present in the above named list, they should not be expected to present functionality and should not be used in a production environment.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/search-the-audit-log-in-security-and-compliance.md:58]:
```suggestion
  > You may see some audited activities available in the Security and Compliance Center Audit log search UI that are not yet functional (such as Forms events). Until the these classifications of events are present in the above named list, they should not be expected to present functionality and should not be used in a production environment.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by mypil** [comment]
@jetkjh - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by markjjo** [comment]
@jetkjh FYI that we are rolling out Microsoft Forms auditing activities and have added them to the documentation:

- https://docs.microsoft.com/en-us/microsoft-365/compliance/search-the-audit-log-in-security-and-compliance#microsoft-forms-activities
- https://docs.microsoft.com/en-us/office/office-365-management-api/office-365-management-activity-api-schema#microsoft-forms-schema

We attempt to keep this documentation as up to date as possible.

**Pull request closed without merging**